src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,202 @@
+        }
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("name"));
+        manufacturer.setCountry(resultSet.getString("country"));
+        Car car = new Car();
+        car.setModel(resultSet.getString("model"));
+        car.setManufacturer(manufacturer);
+        car.setId(resultSet.getObject("car_id", Long.class));
+        return car;
+    }
+
+    private List<Driver> getCarDrivers(Long id) {
+        String query = "SELECT * FROM drivers d "
+                + "JOIN cars_drivers cd ON d.id = cd.driver_id "
+                + "WHERE cd.car_id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getDriversStatement =
+                         connection.prepareStatement(query)) {
+            getDriversStatement.setLong(1, id);
+            ResultSet resultSet = getDriversStatement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                Driver driver = new Driver();
+                driver.setId(resultSet.getObject("id", Long.class));
+                driver.setName(resultSet.getString("name"));
+                driver.setLicenseNumber(resultSet.getString("license_number"));
+                drivers.add(driver);
+            }
+            return drivers;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't find drivers in DB by car id "
+                    + id, throwable);
+        }
+    }
+
+    private Car insertDrivers(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement insertDriversStatement = connection.prepareStatement(query)) {
+            insertDriversStatement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                insertDriversStatement.setLong(2, driver.getId());
+                insertDriversStatement.executeUpdate();
+            }
+            return car;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't insert drivers to car: " + car, throwable);
+        }
+    }
+
+    private boolean deleteDriversFromCar(Car car) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement removeDriverFromCarStatement =
+                         connection.prepareStatement(query)) {
+            removeDriverFromCarStatement.setLong(1, car.getId());
+            removeDriverFromCarStatement.executeUpdate();
+            return true;
```

Comment:
I think we can just return removeDriverFromCarStatement.executeUpdate() >0 here

Reply:
right point